sdk/com.ibm.sbt.web/src/main/webapp/js/sdk/_bridges/dojo-amd/RequestTransport.js

```diff
@@ -55,7 +55,19 @@ define([ "dojo/_base/declare", "dojo/_base/lang", "dojo/request", "../util" ], f
          * 
          */
         request : function(url,options) {
-            return request(url, options);
+        	var promise = new Promise();
+            promise.response = new Promise();
+            request(url, args).response.then(
+                function(response){
```

Comment:
Any particular reason for not using the existing coding convention from Transport.js (Think Mark is pointing to same one in above comment).